libmscore/page.cpp

```diff
@@ -292,15 +293,21 @@ void Page::scanElements(void* data, void (*func)(void*, Element*), bool all)
 
 PageFormat::PageFormat()
       {
-      _size             = QSizeF(210.0/INCH, 297.0/INCH); // A4
-      _evenLeftMargin   = 10.0 / INCH;
-      _oddLeftMargin    = 10.0 / INCH;
-      _printableWidth   = _size.width() - 20.0 / INCH;
-      _evenTopMargin    = 10.0 / INCH;
-      _evenBottomMargin = 20.0 / INCH;
-      _oddTopMargin     = 10.0 / INCH;
-      _oddBottomMargin  = 20.0 / INCH;
+      const int   marginMM  = 10;              // Default margin in millimeters
+      const qreal marginPX  = marginMM * DPMM; //                in pixels/points
+      const qreal marginPX2 = marginPX * 2;    // Double margin  in pixels/points
+
+      _size             = QSizeF(210 * DPMM, 297 * DPMM); // A4 - very hardcoded!!! global preferences
+      _evenLeftMargin   = marginPX;
+      _oddLeftMargin    = marginPX;
+      _printableWidth   = _size.width() - marginPX2;
+// ??? Why no printableHeight???
```

Comment:
I just did that, and eliminated the comment about margins being stored in inches in page.h, as that is no longer the case.  It will be in my next commit.